src/components/TodoList/TodoList.jsx

```diff
@@ -1,4 +1,9 @@
-// Add the required props
-export const TodoList = () => (
-  <>TodoList markup</>
+import { TodoInfo } from '../TodoInfo';
+
+export const TodoList = ({ todos }) => (
```

Comment:
Missing PropTypes for the 'todos' prop, which is required for this component to work properly. It's a good practice to use PropTypes to specify the type of each prop for a component.